packages/builder/src/pages/builder/app/[application]/design/_flagged/index.new.svelte

```diff
@@ -53,27 +69,27 @@
   }
 
   const getContextMenuOptions = (workspaceApp: WorkspaceApp) => {
+    const pause = {
+      icon: workspaceApp.disabled ? "play-circle" : "pause-circle",
+      name: workspaceApp.disabled ? "Activate" : "Disable",
+
+      visible: true,
+      callback: () => {
+        workspaceAppStore.toggleDisabled(
```

Comment:
The behaviour appeared to work as expected here. The app changed to the unpublished state in the UI but I was getting this in the logs.

```js
 err: {
   "type": "TypeError",
   "message": "Cannot read properties of undefined (reading '_id')",
   "stack":
       TypeError: Cannot read properties of undefined (reading '_id')
```

After altering the status of the app, the `Enabled` status in the builder doesn't reflect the current published state and requires a refresh.

Reply:
This was due a backend error. This has been fixed in here: https://github.com/Budibase/budibase/pull/16643
Merging it all so it works as expected 🙏🏽